database/metadata/databases/default/tables/public_location_notes.yaml

```diff
@@ -18,8 +19,9 @@ insert_permissions:
       columns:
         - created_by
         - date
-        - text
+        - is_deleted
         - location_id
+        - text
         - updated_by
         - user_email
```

Comment:
can we delete this column from the notes tables? @johnclary

Reply:
update just found https://github.com/cityofaustin/atd-data-tech/issues/20753 and assigned it to myself, i will do this in a follow up PR :)

Reply:
Thanks for linking to this issue, I was wondering user_email missing from my new notes was a bug